src/main/java/com/example/flight/controllers/AirportController.java

```diff
@@ -0,0 +1,66 @@
+package com.example.flight.controllers;
+
+import java.util.List;
+
+import com.example.flight.entities.Airport;
+import com.example.flight.exceptions.AirportNotFoundException;
+import com.example.flight.repositories.AirportRepository;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+class AirportController {
+
+    private final AirportRepository repository;
```

Comment:
Controllers should not talk directly to the repository. What do you think about this?